reporter-web-app/src/components/LicenseStatsTable.js

```diff
@@ -45,18 +45,7 @@ const LicenseStatsTable = (props) => {
             )(),
             filteredValue: filteredInfo.name || null,
             onFilter: (license, row) => row.name === license,
-            sorter: (a, b) => {
-                const nameA = a.name.toUpperCase();
-                const nameB = b.name.toUpperCase();
-                if (nameA < nameB) {
-                    return -1;
-                }
-                if (nameA > nameB) {
-                    return 1;
-                }
-
-                return 0;
-            },
+            sorter: (a, b) => a.name.localeCompare(b.name),
```

Comment:
If you want to maintain case-insensitive sorting here, you need to keep the `toUpperCase()` calls.

Reply:
I used the `toUpperCase()` as a hack for better performance but I don't need that anymore for `localCompare()`